src/views/Dashboard/Customer/Home/MyTest/StakingServiceTab.vue

```diff
@@ -209,21 +217,11 @@ export default {
     },
 
     setButtonBackground(status) {
-      const colors = Object.freeze({
-        "OPEN": "#F60689",
-        "CLAIMED": "#5640A5",
-        "PROCESSED": "#4CBB65",
-        "WAITINGFORUNSTAKE": "#FAAD15",
-        "UNSTAKED": "#E32319"
-      })
-
-      return { color: colors[status.toUpperCase()] }
+      return STAKE_STATUS_DETAIL.filter(stake => stake.status === status.toUpperCase())[0].color
```

Comment:
this is why I asked this before, because it will take a lot of time to get the specific object we want

Reply:
changed the logic as well. thanks